hail/python/hail/plot/plots.py

```diff
@@ -462,6 +472,9 @@ def frange(start, stop, step):
     data = grouped_ht.filter(hail.is_defined(grouped_ht.x) & (grouped_ht.x != str(x_range[1])) &
                              hail.is_defined(grouped_ht.y) & (grouped_ht.y != str(y_range[1]))).to_pandas()
 
+    if log:
+        data['c'] = np.log10(data['c'])
```

Comment:
Could you have the legend reflect the actual value rather than the log10? Idk how hard that is in bokeh but it would be awesome.